titus-ext/elasticsearch/src/main/java/com/netflix/titus/ext/elasticsearch/ElasticsearchTaskDocumentPublisher.java

```diff
@@ -105,9 +106,9 @@ public ElasticsearchTaskDocumentPublisher(ElasticsearchConfiguration configurati
         PolledMeter.using(registry).withName(MetricConstants.METRIC_ES_PUBLISHER + "errorJsonConversion").monitorValue(errorJsonConversion);
         PolledMeter.using(registry).withName(MetricConstants.METRIC_ES_PUBLISHER + "errorEsClient").monitorValue(errorEsClient);
         PolledMeter.using(registry).withName(MetricConstants.METRIC_ES_PUBLISHER + "errorInPublishing").monitorValue(errorInPublishing);
-        PolledMeter.using(registry)
+        timeSinceLastPublished = PolledMeter.using(registry)
```

Comment:
These `PolledMeter`s should be moved to the `enterActiveMode` so they will only be published for the active leader.